examples/specs/histogram_binned_transform.vl.json

```diff
@@ -10,7 +10,16 @@
   "encoding": {
```

Comment:
I don't think we should rename the file?  
It's a bin transform, not a binned transform.